code/modules/hydroponics/hydroponics.dm

```diff
@@ -532,6 +532,7 @@
 	if(myseed && myseed.loc != src)
 		myseed.forceMove(src)
 	SEND_SIGNAL(src, COMSIG_HYDROTRAY_SET_SEED, new_seed)
+	age = 1
```

Comment:
Looking through, a lot of calls to `set_seed` manually set age after it is called

Some places set age to 0, some set it to 1

I think you should go through and delete places which manually set it after calling set seed so we can consistently go to age 1 or 0